editor/plugins/shader_editor_plugin.cpp

```diff
@@ -770,7 +770,7 @@ void ShaderEditorPlugin::_notification(int p_what) {
 
 ShaderEditorPlugin::ShaderEditorPlugin() {
 	window_wrapper = memnew(WindowWrapper);
-	window_wrapper->set_window_title(vformat(TTR("%s - Godot Engine"), TTR("Shader Editor")));
+	window_wrapper->set_window_title(vformat(TTR("%s - Godot Engine"), TTR("GDShader")));
```

Comment:
This doesn't seem right, the shader editor is used for Visual Shader too. Potentially we *could* add conditional logic so that the window title displays `Shader Editor - GDShader` or `Shader Editor - VisualShader`.